src/lib.rs

```diff
@@ -218,6 +220,7 @@ impl<'a> MarkovChain<'a> {
     ///
     /// [`generate_from`]: struct.MarkovChain.html#method.generate_from
     /// [`iter`]: struct.MarkovChain.html#method.iter
```

Comment:
Could you please add a note about the feature requirement to this and the other methods:

```markdown
**Note:** Only available when the `getrandom` Cargo feature is enabled.
```

Also, I suggest expanding the docstring to explain the alternative in case `getrandom` is not enabled. Something like:

> This method uses `rand::thread_rng()` as its random number generator. Please use `generate_with_rng` instead if you want to use a custom RNG.

Reply:
This became obsolete now that we simply removed the dependency on `thread_rng` and thus also on `getrandom`.